src/chatlist.rs

```diff
@@ -86,7 +86,17 @@ impl Chatlist {
         query: Option<&str>,
         query_contact_id: Option<u32>,
     ) -> Result<Self> {
-        let mut add_archived_link_item = 0;
+        let mut add_archived_link_item = false;
+
+        // drafts are hidden in the database.
+        // if add_drafts is set, this will be expanded to `hidden=1 AND state=DC_STATE_OUT_DRAFT`.
+        // otherwise, this results in `hidden=0 AND state=DC_STATE_OUT_DRAFT` in the sql statement
+        // and no additional messages will be regarded
+        let add_drafts = if 0 != listflags & DC_GCL_ADD_DRAFTS {
```

Comment:
```suggestion
        let add_drafts = (0 != listflags & DC_GCL_ADD_DRAFTS) as u32;
```
see https://doc.rust-lang.org/std/primitive.bool.html